### DISCUSSION TITLE
have one accordion panel to be open by default?

### DISCUSSION THREAD

**Discussion by plweil** [tabpanelwidget/tabpanelwidget#20] [Q&A]
Can we designate one accordion panel (e.g., the first) to be open by default? I don't see anything in the props except maybe selectedIdxs, but that doesn't seem to be it(?)

In tabpanel mode, the first table is open by default...seems as though we ought to be able to do the same in accordion mode, probably via a prop.

I suppose I could programmatically unhide it by the id, but it feels hackish.

**Comment by thierryk** [comment]
Check the [usage section](https://tabpanelwidget.com/#le-code) just before the FAQ:

> **Classes**
> "Out-of-the-box", the Widget will appear as shown in the [Demo section](https://tabpanelwidget.com/#demos) but [a few > classes](https://github.com/tabpanelwidget/tabpanelwidget/wiki#tpw-classes) give you different options.
>
> All classes are meant to be applied to the Widget (the wrapper) with the exception of tpw-selected which, applied to a "heading" (or multiple headings in the case of an Accordion), will let you arbitrarily choose which panel(s) to open by default (the one(s) associated with that/those heading(s)).

**Reply by plweil** [reply]
[edit]
@thierryk I tried doing this programmatically (what a bear of a selector), applied the class to the `<h2>`, and yet the accordion panel remains collapsed. The elements following it still have `aria-expanded="false"` and `hidden`, respectively.

What am I missing here?
-------------------------------------------------

@thierryk I see that, thanks. But -- unless this has to be done programmatically -- if the tabpanel html is being generated by TabPanelWidget, how can I apply this class to a relevant element? I have a vue template, and all I can work with there is the tabpanelwidget component element. E.g.,


```javascript
<template>
  <div>
    <Tabpanelwidget :heading="2" :mode="accordion" :selected-idxs="[0,1,2]" :tabs="['Undergraduate Majors', 'Certificates', 'Degrees']"  animate skin="bar" icon-style="null" disconnected icons-at-the-end >
      <template v-slot:panel-0="">
<!--        override content for panel 0-->
        <div class="uw-tab-content" id="majors-tab-content">
          <div class="uw-row">
            <div class="uw-col col-full-width">
[tab panel content]
```
